packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -348,14 +348,35 @@ export interface IFluidDataStoreContextEvents extends IEvent {
 	(event: "attaching" | "attached", listener: () => void);
 }
 
+//* TODO: Find final home for these
+
+export interface IProvideFluidInternalReferenceInfo {
+	//* Make required here?
+	IFluidInternalReferenceInfo?: IFluidInternalReferenceInfo;
+}
+
+//* TODO: Consider converging this type with UnreferencedStateTracker (e.g. have UnreferencedStateTracker implement this interface)
+/**
+ * Info that may be provided by a Fluid Object regarding whether it's referenced or not by other
+ * objects within this container.
+ */
+export interface IFluidInternalReferenceInfo extends Partial<IProvideFluidInternalReferenceInfo> {
```

Comment:
Maybe `IExperimentalInternalGCInfo`? We should include the term "GC" to make it clear it has to with GC. 
Adding the term "experimental" indicate that this is an experimental functionality which may or may not become permanent - this is what the plan is right?

Reply:
Ok.  I'll do `IExperimentalFluidGCInfo`.  I personally don't think `Experimental` is necessary since the FluidObject / Provider pattern makes it easy to deprecate, but I don't mind either.